rust-src/concordium_base/CHANGELOG.md

```diff
@@ -10,6 +10,11 @@
 - Add `FromStr` and `Display` instances to `dodis_yampolskiy_prf::SecretKey`.
 - Change `Debug` instance of `dodis_yampolskiy_prf::SecretKey` to hide the value.
 
+### Breaking changes
```

Comment:
You've introduced this section now, but a lot of breaking changes have not been included in it.

Since we have not been grouping changes before in the changelog don't just start it like this.